power_grid_model_c/power_grid_model_c/include/power_grid_model_c/dataset.h

```diff
@@ -83,6 +83,47 @@ PGM_API PGM_Idx PGM_dataset_info_elements_per_scenario(PGM_Handle* handle, PGM_D
  */
 PGM_API PGM_Idx PGM_dataset_info_total_elements(PGM_Handle* handle, PGM_DatasetInfo const* info, PGM_Idx component_idx);
 
+/**
+ * @brief Return if a component has attribute indications.
```

Comment:
```suggestion
 * @brief Return whether a component has attribute indications.
```

otherwise, it may suggest that it never returns if not 😂